test/0.8.25/vaults/contracts/ShareRate.t.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../../../contracts/0.8.9/EIP712StETH.sol";
+import "forge-std/Test.sol";
+
+import {CommonBase} from "forge-std/Base.sol";
+import {LidoLocator} from "../../../../contracts/0.8.9/LidoLocator.sol";
+import {StdCheats} from "forge-std/StdCheats.sol";
+import {StdUtils} from "forge-std/StdUtils.sol";
+import {Vm} from "forge-std/Vm.sol";
+import {console2} from "forge-std/console2.sol";
+import {Protocol__Deployment, ILido} from "./Protocol__Deployment.t.sol";
```

Comment:
Maybe better naming is Protocol__Context so as to put methods related to the protocol interactions later, e.g., adding / removing permissions, impersonating agents, etc...